Apps/Admin/Server/Services/IDelegationService.cs

```diff
@@ -28,8 +29,9 @@ public interface IDelegationService
         /// Retrieves delegation information for a person.
         /// </summary>
         /// <param name="phn">The phn to query on.</param>
+        /// <param name="ct">A cancellation token.</param>
         /// <returns>Information about the person and their delegates.</returns>
-        Task<DelegationInfo> GetDelegationInformationAsync(string phn);
+        Task<DelegationInfo> GetDelegationInformationAsync(string phn, CancellationToken ct = default);
```

Comment:
Working hard to enable cancellation token propagation - worth getting the token from the request and pass it along the chain..